src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -564,6 +569,7 @@ namespace wallet_rpc
         KV_SERIALIZE(subaddr_indices)
         KV_SERIALIZE(priority)
         KV_SERIALIZE(mixin)
```

Comment:
Should mixin not be marked as optional now too?

Reply:
If `ring_size` is set then we won't even bother looking at `mixin`, but if it isn't set then it will have the same functionality as before; the mixin will just get adjusted to the default. However, if you all think that it would be preferable, we could make `mixin` optional and set it's default value to the default mixin (4) so that neither is explicitly mandatory.

Reply:
Might as well make it default to 0 too. Default mixin changes with time.